src/styles/main.scss

```diff
@@ -1,3 +1,51 @@
-body {
+@import "./__variables.scss";
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
   margin: 0;
 }
+
+.calendar {
+  box-sizing: border-box;
+  margin: auto;
+  // border: 0.1px solid #000;
+  display: flex;
+  flex-direction: row-reverse;
+  flex-wrap: wrap-reverse;
+  justify-content: end;
+  width: 710px;
+  // height: 510px;
+  gap: 1px;
+}
+
+.cell {
+  @for $num from $month-lehgth through 1 {
+    &:nth-child(#{$num}) {
+      content: "#{($num - $month-lehgth -1 )*-1}";
+      display: flex;
+      box-sizing: border-box;
+      width: 100px;
+      height: 100px;
+      border: 1px solid #000;
+      background-color: #eee;
+      align-items: center;
+      justify-content: center;
+      font-size: 30px;
+    }
+  }
+}
+
+.calendar__day {
```

Comment:
you don't need full classname here, use inside parent selector with &
https://sass-lang.com/documentation/style-rules/parent-selector